src/intents.cs

```diff
@@ -323,7 +323,8 @@ public enum INPaymentStatus : nint {
 		Pending,
 		Completed,
 		Canceled,
-		Failed
+		Failed,
+		Unpaid
```

Comment:
For the future: you can add a comma after the last element in an enum, that way there will be less noise in the diff when you add more elements (since you don't have to modify the last line to add the comma).